### PR TITLE
:bug: Fix pagination issue in /executions - N8N-4236

### PR DESCRIPTION
**Pull request by RicardoE105**
https://community.n8n.io/t/does-pagination-work-for-retrieve-all-executions/16146

### CODE REVIEW

**Comment by RicardoE105** [comment]
Fixes #3834

File: packages/cli/test/integration/publicApi/executions.test.ts [hunk 1]
```diff
@@ -279,6 +279,71 @@ test('GET /executions should retrieve all successfull executions', async () => {
 	expect(waitTill).toBeNull();
 });
 
+test.only('GET /executions should paginate two executions', async () => {
```
Comment (ivov) [inline packages/cli/test/integration/publicApi/executions.test.ts:282]:
```suggestion
test('GET /executions should paginate two executions', async () => {
```

**Review by BHesseldieck — DISMISSED** [review]

**Review by csuermann — APPROVED** [review]
👍

**Review by BHesseldieck — APPROVED** [review]

**Comment by ivov** [comment]
https://linear.app/n8n/issue/N8N-4315/fix-pagination-test-on-postgres-and-mysql

**Comment by janober** [comment]
Got released with `n8n@0.190.0`

**Pull request merged**